pkg/virt-controller/watch/vm/vm.go

```diff
@@ -860,9 +860,11 @@ func (c *Controller) handleVolumeUpdateRequest(vm *virtv1.VirtualMachine, vmi *v
 	case *vm.Spec.UpdateVolumesStrategy == virtv1.UpdateVolumesStrategyMigration:
 		// Validate if the update volumes can be migrated
 		if err := volumemig.ValidateVolumes(vmi, vm, c.dataVolumeStore, c.pvcStore); err != nil {
+			if !errors.Is(err, storagetypes.ErrPVCNotFound) && !errors.Is(err, storagetypes.ErrDVNotFound) {
```

Comment:
`&&` or `||` ?

Reply:
just this bit here, did you mean to use AND?

Reply:
I want to set the required condition only if the error isn't PVC or DV not found. Isn't this correct?:
```
!(err =  ErrPVCNotFound || err =  ErrDVNotFound)  -> !(err =  ErrPVCNotFound) && !(err =  ErrDVNotFound) 
```

Reply:
but the logic is a bit twisted, before I was thinking to return always an error, but now that we are returning here as well, we can simplify the logic, let me change it

Reply:
hopefully, now, it is more readable, we return an error when the PVC or the DV don't exist, otherwise we set the restart condition